schemas/acquisitions/mod-orders-storage/piece.json

```diff
@@ -0,0 +1,168 @@
+      "enumeration": {
+        "type": "string",
+        "description": "Enumeration is the descriptive information for the numbering scheme of a serial. Synchronized with inventory item."
+      },
+      "chronology": {
+        "type": "string",
+        "description": "Chronology is the descriptive information for the dating scheme of a serial. Synchronized with inventory item."
+      },
+      "barcode": {
+        "type": "string",
+        "description": "Barcode is the descriptive information for the barcode of a serial. Synchronized with inventory item."
+      },
+      "accessionNumber": {
+        "type": "string",
+        "description": "AccessionNumber is the descriptive information for the accession number of a serial. Synchronized with inventory item."
+      },
+      "callNumber": {
+        "type": "string",
+        "description": "CallNumber is the descriptive information for the call number of a serial. Synchronized with inventory item."
+      },
+      "discoverySuppress": {
+        "type": "boolean",
+        "description": "Records the fact that the record should not be displayed in a discovery system"
+      },
+      "copyNumber": {
+        "type": "string",
+        "description": "Copy number of the piece"
+      },
+      "receivingStatus": {
+        "description": "The status of this piece",
+        "type": "string",
+        "enum": [
+          "Received",
+          "Expected",
+          "Late",
+          "Claim delayed",
+          "Claim sent",
+          "Unreceivable"
+        ],
+        "default": "Expected"
+      },
+      "supplement": {
+        "description": "Whether or not this is supplementary material",
+        "type": "boolean"
+      },
+      "isBound": {
+        "description": "Whether or not piece has already been bound",
+        "type": "boolean",
+        "default": false
+      },
+      "receiptDate": {
+        "description": "Date that associated item is expected to arrive",
+        "type": "string",
+        "format": "date-time"
+      },
+      "receivedDate": {
+        "description": "The date associated item is actually received",
+        "type": "string",
+        "format": "date-time"
+      },
+      "statusUpdatedDate": {
+        "description": "Date when the status of this piece was last updated",
+        "type": "string",
+        "format": "date-time"
+      },
+      "claimingInterval": {
+        "description": "Time interval (in days) for processing claims related to this piece",
+        "type": "integer"
+      },
+      "internalNote": {
+        "description": "Internal note for Send Claim action",
+        "type": "string"
+      },
+      "externalNote": {
+        "description": "External note for Send Claim action to share with Vendor",
+        "type": "string"
+      },
+      "metadata": {
+        "type": "object",
+        "$ref": "../../common/metadata.json",
+        "readonly": true
+      }
+    },
+    "additionalProperties": false,
```

Comment:
lets remove this field to be flexible to future changes of piece in acq-models

Reply:
Got it, removed